templates/tpos/tpos.html

```diff
@@ -412,13 +432,6 @@ <h5 v-show="!denomIsSats" class="q-mt-none q-mb-sm">
         ></q-page-sticky
       >
     </template>
-    <q-btn
-      v-show="atmMode"
-      color="negative"
-      @click="() => {atmMode = false; getRates()}"
```

Comment:
This is important!! rates may have a premium that needs to be restored to normal when leaving ATM mode!

Reply:
But isn't it taken care of on above line 44?

Reply:
yes